tests/framework/ab_test.py

```diff
@@ -101,6 +106,11 @@ def git_ab_test(
         return result_a, result_b, comparison
 
 
+def is_pr() -> bool:
+    """Returns ``True` iff we are executing in the context of a build kite run on a pull request"""
```

Comment:
```suggestion
    """Returns ``True` if we are executing in the context of a build kite run on a pull request"""
```

Reply:
Ah, removed the extra backtick. The "iff" is correct though ("if and only if" is the logically correct way to phrase these, even though most people dont do it, smh)